interview.md

```diff
@@ -58,11 +78,19 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I would say %50/%50
+
+However, I could create the language seperate from the system and do that later
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I think this is a solid project idea for this course. Not too big. Seems achievable 
```

Comment:
I like this project idea too. One of my DSLs was a notifications manager, and I feel this idea has overlap. I wonder what additional benefits would come from being a language rather than an app. A lot of the ideas seem doable through an app. Also, yes - the scope feels achievable in this class.